pkg/interface/soto/src/js/components/root.js

```diff
@@ -18,19 +18,53 @@ export class Root extends Component {
   render() {
     return (
       <BrowserRouter>
-        <div style={{height: "100vh"}}>
-        <HeaderBar/>
-        <Route exact path="/~dojo" render={ () => {
-          return (
-            <div className="pa3 bg-black mono gray3 w-100"
-            style={{lineHeight: "1.4", height: "calc(100% - 48px)", cursor: "text"}}>
-              <History commandLog={this.state.txt}/>
-              <Input ship={ship} cursor={this.state.cursor} prompt={this.state.prompt} input={this.state.input}/>
-            </div>
-          )}}
-        />
+        <div className="w-100 h-100 bg-gray1">
+          <Route
+            exact path="/~dojo/:popout?"
+            render={(props) => {
+              let popout = !!props.match.params.popout;
+
+              let popoutClasses = classnames({
+                "h-100-m40-p1-ns": !popout,
+                "h-100-m h-100-l h-100-xl": popout,
+                "mh4-m mh4-l mh4-xl": !popout,
+                "mb4-m mb4-l mb4-xl": !popout,
+                "ba-m ba-l ba-xl": !popout
+              })
+
+              return (
+                <div className="w-100 h-100 flex-m flex-l flex-xl">
+                  <div
+                    className="db dn-m dn-l dn-xl inter bg-gray0 dt w-100"
+                    style={{ height: 40 }}>
+                    <a className="f8 pl3 white dtc h-100 v-mid" href="/">
+                      ⟵ Landscape
+                    </a>
+                  </div>
+                  <div
+                    className={
+                      `pa3 bg-gray0 mono white w-100
+                    h-100-m40-s b--gray0 br2 f8 relative ` + popoutClasses
```

Comment:
don't do this style, break it into multiple strings if you need to

Reply:
I have an idea on tidying it up. Standby

Reply:
Better?

Reply:
no. if you're going to incorporate a linebreak, make it two different strings

Reply:
Just want to clarify before proceeding. You would prefer two strings to [a list](https://github.com/urbit/urbit/pull/2201/commits/7babcae532cce3b490c9ad8b91ef2370f82df091#diff-d298c28e00e8174b6b81b342c6cbb437R35)? It seems easier to edit later.

Reply:
yes, leave it inline if possible for styles